src/modules/profile/pages/profileSettings/ProfileSettingsMain.tsx

```diff
@@ -0,0 +1,27 @@
+import { useEffect } from 'react'
+import { useNavigate } from 'react-router-dom'
+
+import { useAuthContext } from '@/context'
+import { getPath } from '@/shared/constants'
+import { useMobileMode } from '@/utils'
+
+import { ProfileSettingsMenuMobile } from './navigation'
+
+export const ProfileSettingsMain = () => {
+  const isMobile = useMobileMode()
+
+  const { user } = useAuthContext()
+  const navigate = useNavigate()
+
+  useEffect(() => {
+    if (!isMobile) {
+      navigate(getPath('userProfileSettingsGeneral', user?.id))
+    }
+  }, [isMobile, navigate, user])
```

Comment:
Make sure to add this check, otherwise, when it is loaded directly from the URL, this could run when the user.id is missing

```suggestion
  useEffect(() => {
    if (!isMobile && user?.id) {
      navigate(getPath('userProfileSettingsGeneral', user?.id))
    }
  }, [isMobile, navigate, user])
```